### ISSUE TITLE
US-NEISO->US-NY is from the future

### ISSUE THREAD

**Issue by systemcatch** [tmrowco/electricitymap#1198]
Goddamn time traveling exchanges!
```
Traceback (most recent call last):
  File "feeder_electricity.py", line 185, in fetch_exchange
    validate_exchange(obj, pair)
  File "/home/electricitymap/parsers/lib/quality.py", line 23, in validate_exchange
    raise Exception("Data from %s can't be in the future" % k)
Exception: Data from US-NEISO->US-NY can't be in the future
```

**Comment by jarek** [comment]
I believe this should be fixed by #1245, let's watch Kibana for a few days and verify

**Comment by jarek** [comment]
I guess it's fixed now, reopen if needed

![](https://user-images.githubusercontent.com/47415/37734755-5a9743b2-2d4c-11e8-909c-1e2053273991.png)
